generators/package-json/package.json

```diff
@@ -14,9 +14,8 @@
   "dependencies": {
     "@manypkg/find-root": "1.1.0",
     "@manypkg/get-packages": "1.1.3",
-    "@types/mri": "1.1.1",
     "@types/node": "16.18.31",
-    "mri": "1.2.0",
+    "cac": "^6.7.14",
```

Comment:
I was wondering if we should use a fixed version as we do with the rest of the dependencies 🤔

Reply:
Ah sure, I can change that. Thanks 🙏